tests/components/vesync/test_config_flow.py

```diff
@@ -0,0 +1,58 @@
+"""Test for vesync config flow."""
+from unittest.mock import patch
+import pyvesync  # pylint: disable=W0611
```

Comment:
Do we need to import `pyvesync`?

Reply:
No, requirements_test_all.txt puts it in the testing environment anyway